java/src/jmri/jmrix/roco/z21/Z21Message.java

```diff
@@ -272,6 +276,12 @@ public String toMonitorString() {
         return toString();
     }
 
+    private String interpretBroadcastFlags(){
+        int flags = getElement(4 ) + (getElement(5) << 8)
+                + (getElement(6) << 16) + (getElement(7) << 24);
+        return "" + flags;
```

Comment:
If I'm reading the Java source code correctly, this line calls `Integer.toString(flags)` and then concatenates that result to an empty String. We probably just want to `return Integer.toString(flags)` instead.

Reply:
fixed. thanks.